package.json

```diff
@@ -27,6 +27,7 @@
     "hot-assets": "(cd client && npm run hot-assets)",
     "jest": "cd client && npm run jest",
     "jest:u": "cd client && npm run jest -- -u",
+    "jest:update": "cd client && npm run jest -- -u",
```

Comment:
Also threw this in here to make our lives a bit easier. I can never remember whether it's `:u` or `:update`, so now both work.